src/js/components/search/filters/location/LocationList.jsx

```diff
@@ -51,7 +51,7 @@ export default class LocationList extends Typeahead {
         this.typeahead.list = this.props.autocompleteLocations;
 
         this.props.autocompleteLocations.forEach((value) => {
-            let key = `<strong>${value.place}</strong><br>${_.upperCase(value.place_type)}`;
+            let key = `<b>${value.place}</b><br>${_.upperCase(value.place_type)}`;
```

Comment:
I think we need to approach this a different way than having HTML in the autocomplete list:
![screen shot 2017-01-09 at 11 50 38 am](https://cloud.githubusercontent.com/assets/3594363/21774856/edb006a4-d661-11e6-8f84-a1c9a2070a65.png)

![screen shot 2017-01-09 at 11 51 32 am](https://cloud.githubusercontent.com/assets/3594363/21774874/0003115c-d662-11e6-9a77-3d3d81262468.png)